OpenRA.Game/Map/MapPreview.cs

```diff
@@ -204,7 +204,7 @@ public InnerData Clone()
 		public string GetLocalisedString(string key, IDictionary<string, object> args = null)
 		{
 			// Prioritise mod level keys.
-			if (modData.Translation.TryGetString(key, out var message, args))
+			if (TranslationProvider.TryGetString(key, out var message, args))
```

Comment:
I suspect that this will prioritise shellmap-defined translation over the map preview, which isn't ideal. Maybe include a flag to force this to only check mod defined translations?

Reply:
Or check against `modData` directly?

Reply:
shellmap is not supposed to have translations, so I doubt it's an issue

Reply:
moddata also no longer has an instance of Translations, same goes with Map. You'd need to get it from TranslationProvider. 

I think a better solution would be to just not use shellmap translations at all? Perhaps a `map.Visibility != MapVisibility.Shellmap` check in TranslationProvider

Reply:
That would break shellmaps if they want to use translations for one reason or another.

Reply:
added TryGetModString method